packages/jaeger-ui/src/components/DependencyGraph/DependencyForceGraph.js

```diff
@@ -13,9 +13,10 @@
 // limitations under the License.
 
 import React, { Component } from 'react';
-import { InteractiveForceGraph, ForceGraphNode, ForceGraphLink } from 'react-vis-force';
+import { InteractiveForceGraph, ForceGraphNode}  from 'react-vis-force';
 import { window } from 'global';
 import { debounce } from 'lodash';
+import {default as JaegerForceGraphArrowLink} from './JaegerForceGraphArrowLink';
```

Comment:
The `{ default as ... }` is not necessary; it can simply be:

```js
import AnyName from './the-file';
// same as:
import { default as AnyName } from './the-file';
```

[Ref](http://exploringjs.com/es6/ch_modules.html#_default-exports-one-per-module)

Reply:
agreed